Sources/MySQL/Internals/Packet.swift

```diff
@@ -81,13 +85,14 @@ internal final class Packet: ExpressibleByArrayLiteral {
     convenience init(arrayLiteral elements: UInt8...) {
         let pointer = MutableBytesPointer.allocate(capacity: 4 &+ elements.count)
         
-        let packetSizeBytes = [
+        let packetSizeAndSequenceIdBytes = [
             UInt8((elements.count) & 0xff),
```

Comment:
Preferentially to adding the (fake) sequence ID to this array, I recommend keeping the original name and memcpy and adding
```swift
var sequenceId = UInt8(0)

memcpy(pointer.advanced(by: 3), &sequenceId, 1)
```
after the original line 90

Reply:
@gwynne can't you just do `pointer[3] = sequenceId` ?

Reply:
Actually, yes!